walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,123 +15,80 @@
 package com.google.sps;
 
 import com.google.common.collect.Sets;
+import com.google.common.collect.Sets.SetView;
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.HashSet;
 import java.util.Iterator;
 import java.util.Set;
+import java.util.TreeSet;
 
 public final class FindMeetingQuery {
 
   /*
-   * Takes in a list of non-overlapping time ranges sorted by start (and end) as well as a new
-   * event. Returns a list of non-overlapping time ranges covered by the existing list as well
-   * as the given event.
+   * Creates a time range of length at least duration from block and its subsequent optional
```

Comment:
At first read, I didn't catch that the combined block is only for subsequent blocks that only have optional attendees. "subsequent optional blocks" is hard to understand here.